Main.sublime-menu

```diff
@@ -6,7 +6,7 @@
                 "id": "package-settings",
                 "children": [
                     {
-                        "caption": "TabNine",
+                        "caption": "tabnine",
```

Comment:
Maybe add the icon thingy here as well?

Reply:
cool 👍 
![image](https://user-images.githubusercontent.com/60742964/99697469-6ef6dd80-2a98-11eb-8c46-e5cd2db5dfdb.png)
![image](https://user-images.githubusercontent.com/60742964/99697517-7ae29f80-2a98-11eb-9b70-47b6c3af7801.png)